source/documentation/retire-packages.html.md

```diff
@@ -0,0 +1,50 @@
+---
+author: ykarel
+title: Retiring a package
+---
+
+# Retiring a package from RDO
+
+1. toc
+{:toc}
+
+### How to remove an OpenStack package from RDO Trunk
+
+Package for a project can exist across different releases. So retiring it in RDO needs to go via stages,
+To remove a package, following steps are required:-
+
+Stage 1:- Project Source is retired/deleted upstream, but it's package still need to be maintained because it's required by other projects
+
+- Send a review to `rdoinfo` like [Example review](https://review.rdoproject.org/r/#/c/20387/) to pin(add source-branch: `<good commit>`) the project against the tag or commit, so package get's build from pinned commit rather than latest commit(deleted source code).
+
+    ```
+    # Need to add source-branch like below
+    - project: oslo-sphinx
+      tags:
+        train-uc:
+          source-branch: f92583cfc34292ec1441368f984c9692346946c4
+     ```
+
+- Send a review to `config` project like [Example review](https://review.rdoproject.org/r/#/c/20415/) to run DLRN-pinned jobs(legacy-DLRN-rpmbuild-pinned, legacy-DLRN-rpmbuild-fedora-pinned) so package get's build from pinned commit rather than from latest, This is required so that spec changes can be done like https://review.rdoproject.org/r/#/c/20383/.
+
+Stage 2:- Package is no longer needed by other projects:-
+
+- Send a review to [rdoinfo project in review.rdoproject.org](https://review.rdoproject.org/r/#/q/project:rdoinfo).
+In this change you need to delete the tag for the project for which package is not needed to built.
+    
+Once the change is merged in rdoinfo, package will no longer exist in RDO trunk repo, but it will still be built for previous releases(for which
+tag is defined in `rdoinfo`):-
+[RDO Trunk repos](http://trunk.rdoproject.org/centos7-master/report.html).
+
+Stage 3:- Package is no longer needed as it's not built for any release, no spec changes needed in future, so project needs to be deleted from review.rdoproject.org.
+
+- Send a review to `config` project to remove all jobs against the project and just add noop job like below so patches can be merged in the projects as Zuul +1 is required:
+
+    ```
+    # following need to be added to the project's job config to run noop jobs:-
+    - templates:
+        - noop
+    ```
+
+- Delete the project spec files and add dead.package file mentioning the reason of removal
+- Remove project completely(project-config, resource,gerritbot etc) from zuul config
```

Comment:
We can't remove the resource, project-config, etc... as we still maintain it for stable releases.

Reply:
hmm this step is only valid when package is no longer build and required for any releases. Low chance for such a case though.